app/main.py

```diff
@@ -2,0 +1,19 @@
+from __future__ import annotations
+
+
+def calculate_team_total_rating(players: list) -> int:
+    result = 0
+
+    for player in players:
+        result += player.get_rating()
+    return result
+
+
+def elves_concert(players: list) -> str:
+    for player in players:
+        player.play_elf_song()
+
+
+def feast_of_the_dwarves(players: list) -> str:
```

Comment:
The function `feast_of_the_dwarves` is declared to return a `str`, but it currently does not return any value. Consider returning a string or changing the return type to `None`.